package.json

```diff
@@ -50,7 +50,6 @@
         "@typescript-eslint/parser": "^4.9.1",
         "babel-loader": "^8.2.2",
         "css-loader": "^5.0.1",
-        "dotenv": "^8.2.0",
```

Comment:
How come this was removed?

Reply:
Already talked to @brandonwong368 about this but let's also remove `package-lock.json`